contracts.json

```diff
@@ -81,39 +81,33 @@
     "NCCR_V0": {
       "proxyAddress": "0xBBbD7AEBD29360a34ceA492e012B9A2119DEd306"
     },
-    "Nori_V0": {
-      "proxyAddress": "0x1f77C0415bc4E5B5Dcb33C796F9c8cd8cc1c259d"
-    },
     "NORI": {
-      "proxyAddress": "0x961760Ad1bEd52bf4d79aa4b1558E7F9d72071e4"
+      "proxyAddress": "0x0000000000000000000000000000000000000000"
```

Comment:
is mainnnet NORI supposed to be zeroed out liek this?

Reply:
i guess just indicating that this version of NORI is not yet actually deployed on mainnet?